File: chakhaeng-backend/src/main/java/com/ssafy/chakeng/stats/dto/ViolationStatistics.java
package com.ssafy.chakeng.stats.dto;

import java.util.List;

public record ViolationStatistics(
        int totalDetections,
        int detectionAccuracy,         // %
        int weeklyDetections,
        double dailyAverageDetections,
        List<ViolationTypeStatistic> violationTypeStats,
        List<HourlyStatistic> hourlyStats,
        List<MonthlyTrend> monthlyTrend
) {}

File: chakhaeng-android/app/src/main/java/com/sos/chakhaeng/domain/model/statistics/ViolationStatistics.kt
package com.sos.chakhaeng.domain.model.statistics

data class ViolationStatistics(
    val totalDetections: Int,
    val detectionAccuracy: Int, // 정확도 (%)
    val weeklyDetections: Int,
    val dailyAverageDetections: Double,

    val violationTypeStats: List<ViolationTypeStatistic>,
    val hourlyStats: List<HourlyStatistic>,
    val monthlyTrend: List<MonthlyTrend>
)
